salt/pillar/git_pillar.py

```diff
@@ -365,12 +365,9 @@
 from salt.exceptions import FileserverConfigError
 from salt.pillar import Pillar
 
-# Import third party libs
-from salt.ext import six
-
-PER_REMOTE_OVERRIDES = ('env', 'root', 'ssl_verify', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'mountpoint', 'all_saltenvs')
-GLOBAL_ONLY = ('base', 'branch')
+PER_REMOTE_OVERRIDES = ('base', 'env', 'root', 'ssl_verify', 'refspecs', 'fallback', 'saltenv_whitelist', 'saltenv_blacklist', 'disable_saltenv_mapping', 'ref_types,)
```

Comment:
This line end by 'ref_types,) which cause a syntax error

Reply:
Yes a single quote is missing: `'ref_types,)`.

Question: are you using python2 or python3?

Reply:
python 2, we still have some old server without python3 to manage, this is why i don't upgrade right up to 3001